domain/generator/src/main/kotlin/com/few/generator/core/Scrapper.kt

```
@@ -0,0 +1,129 @@
+            val sentences = processedText.split("|").map { it.trim() }
+            texts.addAll(sentences.filter { isValidSentence(it) && it.length >= 10 })
+        }
+
+        return texts.distinct().filter { it.split(" ").size >= 4 }
+    }
+
+    fun isValidImageUrl(url: String): Boolean =
+        try {
+            val uri = URI(url)
+            imageExtensions.any { uri.path.lowercase().endsWith(it) }
+        } catch (e: Exception) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)